src/studio/src/designer/frontend/ux-editor/components/config/EditModalContent.tsx

```diff
@@ -753,6 +763,138 @@ export class EditModalContentComponent extends React.Component<IEditModalContent
         );
       }
 
+      case ComponentTypes.FileUploadWithTag: {
```

Comment:
This file is already huge, so would be good to avoid making it even bigger 😁

Could you please extract this to a separate file? See ComponentTypes.Image as an example.

Reply:
Yes, done.